installation.rst

```diff
@@ -48,14 +46,51 @@ Install Conan:
 Known installation issues with pip
 ++++++++++++++++++++++++++++++++++
 
-- When Conan is installed with :command:`pip install --user <username>`, usually a new directory is created for it. However, the directory
-  is not appended automatically to the `PATH` and the :command:`conan` commands do not work. This can usually be solved restarting the session of
-  the terminal or running the following command:
+When Conan is installed with :command:`pip install --user <username>`, usually a new directory is created for it. However, the directory is not appended automatically to the `PATH` and the :command:`conan` commands do not work. This can usually be solved restarting the session of the terminal or running the following command:
 
   .. code-block:: bash
 
       $ source ~/.profile
 
+
+.. _conan_update:
+
+Update
+++++++
+
+If installed via ``pip``, Conan version can be updated with:
+
+.. code-block:: bash
+
+    $ pip install conan --pre --upgrade  # Might need sudo or --user
+
+The upgrade shouldn't affect the installed packages or cache information. If the cache becomes inconsistent somehow, you may want to remove its content by deleting it (``<userhome>/.conan2``).
+
+
+Use system installer or create a self contained executable
+----------------------------------------------------------
+
+There are existing installers in `Conan downloads`_ for OSX Brew, Debian, Windows, Linux Arch, that will not require Python first.
+
+If there is no installer for your platform, you can create your own Conan executable, with the ``pyinstaller.py`` utility in the repo. This proces is able to create a self contained Conan executable that contains all it needs,
+including the Python interpreter, so it wouldnt be necessary to have Python installed in the system.
+
+You can do it with: 
+
+.. code-block:: bash
+
+  $ git clone https://github.com/conan-io/conan conan_src
+  $ cd conan_src
+  $ git checkout develop2 # or to the specific tag you want to
+  $ pip install -e . 
+  $ python pyinstaller.py
+
+
+It is important to install the dependencies and the project first with ``pip install -e .`` which configures the project as "editable", that is, to run from the current source folder. After creating the executable, it can be pip uninstalled.
```

Comment:
```suggestion
It is important to install the dependencies and the project first with ``pip install -e .`` which configures the project as "editable", that is, to run from the current source folder. After creating the executable, it can be uninstalled with pip.
```